lib/rules/constructor-super.js

```diff
@@ -92,6 +92,8 @@ function isPossibleConstructor(node) {
 
 module.exports = {
     meta: {
+        type: "suggestion",
```

Comment:
This seems like it should be a `problem`; the patterns reported by this rule will generally throw an error at runtime.

Reply:
Agreed